src/utils/hooks/useAutofetcher.ts

```diff
@@ -2,7 +2,11 @@ import {DependencyList, useEffect, useRef} from 'react';
 
 import {AutoFetcher} from '../autofetcher';
 
-export const useAutofetcher = (fetchData: VoidFunction, deps: DependencyList, enabled = true) => {
+export const useAutofetcher = (
+    fetchData: (isBackground: boolean) => void,
```

Comment:
In Healthcheck we pass to autoFetcher fetchData without any params.

Is it true, that for TS it is valid to pass function without arguments when function with arguments is expected? Will it be OK to pass isBackground flag to function that does not expect it?

Reply:
Yea, it's fine. This signature makes sure the call expression passes the required arguments into the function. It is not mandatory for the function to accept and use them.